drivers/mmcsd/sdio.c

```diff
@@ -75,7 +75,7 @@ begin_packed_struct struct sdio_resp_r5
     uint32_t io_current_state : 2;
     uint32_t illegal_command  : 1;
     uint32_t com_crc_error    : 1;
-  } flags;
+  } end_packed_struct flags;
```

Comment:
should we move to the next line to avoid the nxstyle warning?

Reply:
Yes, sorry about forgetting the nxstyle. Fixed